### DISCUSSION TITLE
Buttons not working and all lights on after starting flight

### DISCUSSION THREAD

**Discussion by lskix** [flybywiresim/aircraft#2371] [Help]
Just installed the mod, when i load up a flight with it, the cockpit buttons and switches do not work, and all lights are illuminated as if in a christmas tree.

**Comment by ImenesFBW** [comment]
Seems like a bad install or a conflict with another mod in your community folder.

Try using the installer. https://github.com/Externoak/A32NX-installer/releases/latest/download/A32NX_Downloader.zip

**Reply by lskix** [reply]
that was my thought, but it's a fresh install of MSFS and i did use the installer

**Comment by NathanInnes** [comment]
Are you using any other mods that could be conflicting?

**Reply by lskix** [reply]
i'm not, no

**Comment by ghost** [comment]
Do you have the very latest version of the game?
